build.sbt

```diff
@@ -36,6 +36,8 @@ addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "0.8.0" excludeAll(
   ExclusionRule(organization = "com.danieltrinh")
 ))
 
+// If you change the scalariform version, you may also need to generate a new
+// scalariform.jar in src/main/resources/autoformat/
```

Comment:
You ought to be able to pull this out of the ivy cache - but good luck figuring out how! :)

Maybe add a TODO?